modules/claims_api/app/swagger/claims_api/description/v2.md

```diff
@@ -1,6 +1,6 @@
 ## Background
 
```

Comment:
I did not modify the file. This came in from the merge conflict and extra spaces were removed.